osf-packager/src/test/resources/json/IpmPackagerTest/testCreatePackageWithWiki/nodes/ng9em/comments/index.json

```diff
@@ -1,142 +0,0 @@
-{
```

Comment:
Yes, these files are being moved to another project because the test is being moved.  What I meant about not being sure why the files were removed is that they were removed in the repo two months ago, though they were still on my computer.  However, the wiki test (in its new location) seems to run fine as far as the model loading goes.